PowerShellGet/PSModule.psm1

```diff
@@ -8182,6 +8591,7 @@ function Publish-PSArtifactUtility
         if($PSModuleInfo.PrivateData -and 
            ($PSModuleInfo.PrivateData.GetType().ToString() -eq "System.Collections.Hashtable") -and 
            $PSModuleInfo.PrivateData["PSData"] -and
+           $PSModuleInfo.PrivateData.PSData -and

```

Comment:
>$PSModuleInfo.PrivateData.PSData -and [](start = 11, length = 37)

This line is not required as it is equivalent to above line

$PSModuleInfo.PrivateData["PSData"] -and

Reply:
It looks like "$PSModuleInfo.PrivateData.PSData -and" line is not removed.

---
In reply to: [141265801](https://github.com/rebro-msft/PowerShellGet/pull/2#discussion_r141265801) [](ancestors = 141265801)

Reply:
You are right, I must have missed that.

---
In reply to: [141941815](https://github.com/rebro-msft/PowerShellGet/pull/2#discussion_r141941815) [](ancestors = 141941815,141265801)